composeApp/src/commonMain/kotlin/org/ooni/probe/data/models/InstalledTestDescriptorModel.kt

```diff
@@ -24,6 +28,26 @@ import org.ooni.probe.shared.hexToColor
 import org.ooni.probe.shared.stringMonthArrayResource
 import org.ooni.probe.shared.toEpoch
 
+enum class OoniTest(
+    val id: Long,
+    val key: String,
+) {
+    WEBSITES(10470L, "websites"),
+    INSTANT_MESSAGING(10471L, "instant_messaging"),
+    CIRCUMVENTION(10472L, "circumvention"),
+    PERFORMANCE(10473L, "performance"),
+    EXPERIMENTAL(10474L, "experimental"),
```

Comment:
We're using regular capitalization for enum and sealed classes in this project: WEBSITES -> Websites.